udata/commands/init.py

```diff
@@ -21,7 +21,7 @@ def init():
     generate_fixtures()
 
     log.info('Apply DB migrations if needed')
-    migrate(record=True)
+    migrate(record=True, dryrun=False)
```

Comment:
I think defaulting to False into the upstream `migrate` might be more useful (to avoid this error in other futur usages)